pom.xml

```diff
@@ -93,6 +96,9 @@
         </profile>
         <profile>
             <id>neo4j-5</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
             <properties>
                 <neo4j.version>5.4.0</neo4j.version>
```

Comment:
should we also update this to be `5.6.0` or soon `5.7.0`?

Reply:
I think so, I reduced the PR to the bare minimum but happy to include this as well

Reply:
actually, I forgot I did that here: https://github.com/neo4j-contrib/neo4j-spark-connector/pull/461/ ^^